### PR TITLE
Release Note for WebView2 with Stable 101 and prerelease 102

### PR DESCRIPTION
**Pull request by imicy**
Placeholder # needs to be updated once doc is alive

**Rendered sections for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1930#10121039

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -32,9 +32,58 @@ For more information, see [Matching the Runtime version with the SDK version](co
...
+### General
+
+#### Promotions
+
+*	Add [ServerCertificateErrorDetected API](#) which allows you to handle the event when the WebView2 control cannot verify server's digital certificate.
+* 	The [ClearBrowsingData API](#):
+	*  `clearBrowsingDataInTimeRange`
+	*  `clearBrowsingDataAll`
+
+#### Bug Fixes
```
Comment (vchapel) [inline microsoft-edge/webview2/release-notes.md:77]: @nishitha-burman  Need GitHub issue numbers and links for bug fixes.

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -32,9 +32,58 @@ For more information, see [Matching the Runtime version with the SDK version](co
...
+
+<!-- ====================================================================== -->
 
+## 1.0.1210.39
+  
+Release Date: May 6, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1210.39](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1210.39)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime 101.1210.39 or higher.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:45]:
```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 101.1210.39 or higher.
```
for template consistency

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -32,9 +32,58 @@ For more information, see [Matching the Runtime version with the SDK version](co
...
+
+### General
 
+#### Promotions
+
+The following items are now stable:
+
+*   Support for [multiple user profiles](#) in WebView2.
+
+*	[Theming API](#) which provides a way to customize the WebView2 color theme as `light`, `dark`, or `system`.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:55]: Recommend Writer or template Dev normalize/reduce the number of leading spaces between asterisk and body of each list item.  Linter-like pass.  Markdown likes consistent number of leading spaces within a given .md file.  

Recommend full-text search " \n" with Regex On, in VS Code, to remove trailing spaces from lines.  per periodic cleanup of markdown tagging.

Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:55]: fixed

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md]: fixed

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+## 1.0.1248-prerelease
+  
+Release Date: May 6, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1248-prelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1248-prerelease)  
+  
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 102.0.1248.0 or higher.  
+
+### General
+
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:69]:
```suggestion

* Added support for WinRT Object projection into JavaScript by adding WinRT JS Proj tool in NuGet package. Refer to (full doc)[https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/winrt-from-js?branch=pr-en-us-1910&tabs=csharp] for instructions on how to use the tool.  

```

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+#### Promotions
+
+* Added the ServerCertificateErrorDetected API which allows you to handle an event when the WebView2 control cannot verify the server's digital certificate.
```
Comment (nishitha-burman) [inline microsoft-edge/webview2/release-notes.md:72]:
```suggestion
* The [Server Certificate API](/microsoft-edge/webview2/reference/win32/icorewebview2_14?view=webview2-1.0.1248-prerelease) which provides an option to trust the server's TLS certificate at the application level and render the page without prompting the user about TLS or providing the ability to cancel the web request.

```

**Review by nishitha-burman — DISMISSED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+#### Promotions
+
+The following items are now stable:
+
+* Support for multiple user profiles in WebView2.
+
+* Theming API which provides a way to customize the WebView2 color theme as `light`, `dark`, or `system`.
+
+* Default Download API which provides a way to customize the default download location.
```
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:57]: changed

File: microsoft-edge/webview2/release-notes.md [hunk 7]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+* Support for multiple user profiles in WebView2.
+
+* Theming API which provides a way to customize the WebView2 color theme as `light`, `dark`, or `system`.
```
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:55]: changed

File: microsoft-edge/webview2/release-notes.md [hunk 8]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 101.1210.39 or higher.
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+* Support for multiple user profiles in WebView2.
```
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:53]: changed

File: microsoft-edge/webview2/release-notes.md [hunk 9]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+Release Date: May 6, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1248-prelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1248-prerelease)  
+  
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 102.0.1248.0 or higher.  
+
+### General
+
+#### Promotions
+
```
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:71]: added

File: microsoft-edge/webview2/release-notes.md [hunk 10]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+* Added the ServerCertificateErrorDetected API which allows you to handle an event when the WebView2 control cannot verify the server's digital certificate.
+* The ClearBrowsingData API:
```
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:73]: changed

File: microsoft-edge/webview2/release-notes.md (hunk 5 above)
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:72]: changed

File: microsoft-edge/webview2/release-notes.md (hunk 4 above)
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:69]: added

File: microsoft-edge/webview2/release-notes.md [hunk 11]
```diff
@@ -34,6 +34,56 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+#### Bug Fixes
+  
+* Fixed an unavoidable crash that occurred in the WPF control's `OnWindowPositionChanged` event.  
+* Fixed the issue with `CoreWebView2EnvironmentOptions.ExclusiveUserDataFolderAccess` isn't working properly in .NET SDK.
+* Fixed a runtime regression that caused some Office add-ins which use host objects to crash during operations that previously worked.
+* Fixed an issue where WebView2 content can become blurry when moving between monitors with different scaling.
+* Starting with runtime v102, if you set `ZoomFactor` to the maximum of 5, then set it higher like 6, the value of `ZoomFactor` will be correctly returned as 5 instead of 6.
+* Fixed a regression to make sure that WebView2 creation fails quickly with `HRESULT_FROM_WIN32(ERROR_INVALID_STATE)` instead of time out.
+* Fixed a bug where changes from Chromium broke WebView2 background color.
```
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/webview2/release-notes.md:85]: changed

**Review by vchapel — APPROVED** [review]
LGTM

**Review by nishitha-burman — APPROVED** [review]

**Pull request merged**